airbyte-config/init/create_mount_directories.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env sh
+
```

Comment:
comment explains the operation that we're trying to carry out here. this used to be handled by mounting a volume like so `${LOCAL_ROOT}/..:/local`. compose changes make it so that the `/..` don't get executed anymore, so we have to do all of this chicannery.